jenkins-integration/beaker/install/pe/10_install_pe.rb

```diff
@@ -0,0 +1,3 @@
+test_name 'Install PE'
+
+install_pe
```

Comment:
Yeah, by environment variables or beaker host file config options. The readme in the `install` dir (in this PR) has some more info